applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py

```diff
@@ -50,6 +50,33 @@ def __init__(self, model, project_parameters):
 
         super(StructuralMechanicsAnalysis, self).__init__(model, project_parameters)
 
+    def Initialize(self):
+        """ Initializing the Analysis """
+        super(StructuralMechanicsAnalysis, self).Initialize()
+        self._GetSolver().SetEchoLevel(self.echo_level)
+        # To avoid many prints
+        if self.echo_level == 0:
+            KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING)
```

Comment:
Hm this is ok for the tests but too restrictive for general
How abt an additional setting where you can specify the severity directly?

Reply:
I basically do that because we print many things that in general no one is interested (like the building times), this destroys  the tables of outputs